Add Polish translation.

SzczurekYT opened the pull request:
As I said I finally did it. New language joins the party.

SzczurekYT committed:
Initial translation

SzczurekYT committed:
Godot stuff (translation)

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_TITLE_NEW_PROJECT": {
     "en": "New project",
-    "ja": "新しいプロジェクト",
+    "ja": "\u65b0\u3057\u3044\u30d7\u30ed\u30b8\u30a7\u30af\u30c8",
```
```suggestion
    "ja": "新しいプロジェクト",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Un nouveau projet",
     "nl": "Nieuw project",
-    "zh_TW": "新專案"
+    "zh_TW": "\u65b0\u5c08\u6848",
```
```suggestion
    "zh_TW": "新專案",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_TITLE_RECENT_PROJECTS": {
     "en": "Recent projects",
-    "ja": "最近のプロジェクト",
+    "ja": "\u6700\u8fd1\u306e\u30d7\u30ed\u30b8\u30a7\u30af\u30c8",
```
```suggestion
    "ja": "最近のプロジェクト",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_BUTTON_NEW_FHD_V_PROJECT": {
     "en": "New 1080p (portrait)",
-    "ja": "新1080p　（縦）",
+    "ja": "\u65b01080p\u3000\uff08\u7e26\uff09",
```
```suggestion
    "ja": "新1080p　（縦）",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_BUTTON_NEW_4K_H_PROJECT": {
     "en": "New 4K (landscape)",
-    "ja": "新４K　（横）",
+    "ja": "\u65b0\uff14K\u3000\uff08\u6a2a\uff09",
```
```suggestion
    "ja": "新４K　（横）",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_BUTTON_NEW_4K_V_PROJECT": {
     "en": "New 4K (portrait)",
-    "ja": "新４K　（縦）",
+    "ja": "\u65b0\uff14K\u3000\uff08\u7e26\uff09",
```
```suggestion
    "ja": "新４K　（縦）",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_BUTTON_OPEN_PROJECT": {
     "en": "Open project",
-    "ja": "プロジェクトをを開く",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u3092\u3092\u958b\u304f",
```
```suggestion
    "ja": "プロジェクトをを開く",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_BUTTON_SUPPORT_PROJECT": {
     "en": "Support this project",
-    "ja": "このプロジェクトを支援する",
+    "ja": "\u3053\u306e\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u3092\u652f\u63f4\u3059\u308b",
```
```suggestion
    "ja": "このプロジェクトを支援する",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "STARTUP_PANEL_TITLE": {
     "en": "Welcome!",
-    "ja": "ようこそ！",
+    "ja": "\u3088\u3046\u3053\u305d\uff01",
```
```suggestion
    "ja": "ようこそ！",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Ajouter un nouveau projet",
     "nl": "Nieuw project",
-    "zh_TW": "建立新專案"
+    "zh_TW": "\u5efa\u7acb\u65b0\u5c08\u6848",
```
```suggestion
    "zh_TW": "建立新專案",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Nom du projet :",
     "nl": "Project naam:",
-    "zh_TW": "專案名稱："
+    "zh_TW": "\u5c08\u6848\u540d\u7a31\uff1a",
```
```suggestion
    "zh_TW": "專案名稱：",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "NEW_PROJECT_LOCATION": {
     "en": "Location:",
-    "ja": "保存場所：",
+    "ja": "\u4fdd\u5b58\u5834\u6240\uff1a",
```
```suggestion
    "ja": "保存場所：",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Localisation :",
     "nl": "Locatie:",
-    "zh_TW": "專案位置："
+    "zh_TW": "\u5c08\u6848\u4f4d\u7f6e\uff1a",
```
```suggestion
    "zh_TW": "專案位置：",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Create",
-    "ja": "作成",
-    "fr": "Créer",
+    "ja": "\u4f5c\u6210",
```
```suggestion
    "ja": "作成",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "作成",
-    "fr": "Créer",
+    "ja": "\u4f5c\u6210",
+    "fr": "Cr\u00e9er",
```
```suggestion
    "fr": "Créer",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "TEXT_VERSION": {
     "en": "Version",
-    "ja": "バージョン",
+    "ja": "\u30d0\u30fc\u30b8\u30e7\u30f3",
```
```suggestion
    "ja": "バージョン",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Version",
     "nl": "Version",
-    "zh_TW": "版本"
+    "zh_TW": "\u7248\u672c",
```
```suggestion
    "zh_TW": "版本",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "TEXT_SORT": {
     "en": "Sort:",
-    "ja": "並び替え：",
+    "ja": "\u4e26\u3073\u66ff\u3048\uff1a",
```
```suggestion
    "ja": "並び替え：",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Trier :",
     "nl": "Sorteer:",
-    "zh_TW": "排序"
+    "zh_TW": "\u6392\u5e8f",
```
```suggestion
    "zh_TW": "排序",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "TEXT_FILTER_PROJECTS": {
     "en": "Filter projects",
-    "ja": "プロジェクトのフィルター",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306e\u30d5\u30a3\u30eb\u30bf\u30fc",
```
```suggestion
    "ja": "プロジェクトのフィルター",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Langue",
     "nl": "Taal",
-    "zh_TW": "語言"
+    "zh_TW": "\u8a9e\u8a00",
```
```suggestion
    "zh_TW": "語言",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "SETTING_TEXT_LANGUAGE": {
     "en": "Language",
-    "ja": "言語",
+    "ja": "\u8a00\u8a9e",
```
```suggestion
    "ja": "言語",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "選択",
-    "fr": "Sélectioner",
+    "ja": "\u9078\u629e",
+    "fr": "S\u00e9lectioner",
```
```suggestion
    "fr": "Sélectioner",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Select",
-    "ja": "選択",
-    "fr": "Sélectioner",
+    "ja": "\u9078\u629e",
```
```suggestion
    "ja": "選択",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_CLOSE": {
     "en": "Close",
-    "ja": "閉じる",
+    "ja": "\u9589\u3058\u308b",
```
```suggestion
    "ja": "閉じる",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_FOLDER_ERROR": {
     "en": "This Folder does not exist!",
-    "ja": "このフォルダは存在しません！",
+    "ja": "\u3053\u306e\u30d5\u30a9\u30eb\u30c0\u306f\u5b58\u5728\u3057\u307e\u305b\u3093\uff01",
```
```suggestion
    "ja": "このフォルダは存在しません！",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_PROJECT_FOLDER_BUTTON": {
     "en": "Project folder",
-    "ja": "プロジェクトのフォルダー",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306e\u30d5\u30a9\u30eb\u30c0\u30fc",
```
```suggestion
    "ja": "プロジェクトのフォルダー",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_RETURN_BUTTON": {
     "en": "Back",
-    "ja": "戻る",
+    "ja": "\u623b\u308b",
```
```suggestion
    "ja": "戻る",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_ADD_PATH": {
     "en": "Add path",
-    "ja": "パスの追加",
+    "ja": "\u30d1\u30b9\u306e\u8ffd\u52a0",
```
```suggestion
    "ja": "パスの追加",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Filtrer les projets",
     "nl": "Projecten filteren",
-    "zh_TW": "篩選專案"
+    "zh_TW": "\u7be9\u9078\u5c08\u6848",
```
```suggestion
    "zh_TW": "篩選專案",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "TAB_PROJECTS": {
     "en": "Projects",
-    "ja": "プロジェクト",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8",
```
```suggestion
    "ja": "プロジェクト",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Projets",
     "nl": "Projecten",
-    "zh_TW": "專案"
+    "zh_TW": "\u5c08\u6848",
```
```suggestion
    "zh_TW": "專案",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "About",
-    "ja": "情報",
-    "fr": "À Propos",
+    "ja": "\u60c5\u5831",
```
```suggestion
    "ja": "情報",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "情報",
-    "fr": "À Propos",
+    "ja": "\u60c5\u5831",
+    "fr": "\u00c0 Propos",
```
```suggestion
    "fr": "À Propos",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
+    "fr": "\u00c0 Propos",
     "nl": "Info",
-    "zh_TW": "關於"
+    "zh_TW": "\u95dc\u65bc",
```
```suggestion
    "zh_TW": "關於",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Settings",
-    "ja": "設定",
-    "fr": "Réglages",
+    "ja": "\u8a2d\u5b9a",
```
```suggestion
    "ja": "設定",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
+    "fr": "R\u00e9glages",
     "nl": "Instellingen",
-    "zh_TW": "設定"
+    "zh_TW": "\u8a2d\u5b9a",
```
```suggestion
    "zh_TW": "設定",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "SORT_NAME": {
     "en": "name",
-    "ja": "名前",
+    "ja": "\u540d\u524d",
```
```suggestion
    "ja": "名前",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "nom",
     "nl": "naam",
-    "zh_TW": "名稱"
+    "zh_TW": "\u540d\u7a31",
```
```suggestion
    "zh_TW": "名稱",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "last edited",
-    "ja": "編集日",
-    "fr": "dernier édité",
+    "ja": "\u7de8\u96c6\u65e5",
```
```suggestion
    "ja": "編集日",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
+    "fr": "dernier \u00e9dit\u00e9",
     "nl": "Laatst bewerkt",
-    "zh_TW": "最近修改"
+    "zh_TW": "\u6700\u8fd1\u4fee\u6539",
```
```suggestion
    "zh_TW": "最近修改",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "creation",
-    "ja": "作った日",
-    "fr": "création",
+    "ja": "\u4f5c\u3063\u305f\u65e5",
```
```suggestion
    "ja": "作った日",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "作った日",
-    "fr": "création",
+    "ja": "\u4f5c\u3063\u305f\u65e5",
+    "fr": "cr\u00e9ation",
```
```suggestion
    "fr": "création",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "BUTTON_REMOVE_PROJECT": {
     "en": "Remove project",
-    "ja": "プロジェクトを消す",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u3092\u6d88\u3059",
```
```suggestion
    "ja": "プロジェクトを消す",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Supprimer un projet",
     "nl": "Verwijder project",
-    "zh_TW": "移除專案"
+    "zh_TW": "\u79fb\u9664\u5c08\u6848",
```
```suggestion
    "zh_TW": "移除專案",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "BUTTON_REMOVE_MISSING": {
     "en": "Remove missing projects",
-    "ja": "不足プロジェクトを削除",
+    "ja": "\u4e0d\u8db3\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u3092\u524a\u9664",
```
```suggestion
    "ja": "不足プロジェクトを削除",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "fr": "Supprimer les projets manquants",
     "nl": "Verwijder vermiste projecten",
-    "zh_TW": "移除缺失專案"
+    "zh_TW": "\u79fb\u9664\u7f3a\u5931\u5c08\u6848",
```
```suggestion
    "zh_TW": "移除缺失專案",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Select Project Folder",
-    "ja": "プロジェクトフォルダを選択してください",
-    "fr": "Sélectionner le Dossier du Projet",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u30d5\u30a9\u30eb\u30c0\u3092\u9078\u629e\u3057\u3066\u304f\u3060\u3055\u3044",
```
```suggestion
    "ja": "プロジェクトフォルダを選択してください",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "プロジェクトフォルダを選択してください",
-    "fr": "Sélectionner le Dossier du Projet",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u30d5\u30a9\u30eb\u30c0\u3092\u9078\u629e\u3057\u3066\u304f\u3060\u3055\u3044",
+    "fr": "S\u00e9lectionner le Dossier du Projet",
```
```suggestion
    "fr": "Sélectionner le Dossier du Projet",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
+    "fr": "S\u00e9lectionner le Dossier du Projet",
     "nl": "Selecteer project map",
-    "zh_TW": "選擇專案位置"
+    "zh_TW": "\u9078\u64c7\u5c08\u6848\u4f4d\u7f6e",
```
```suggestion
    "zh_TW": "選擇專案位置",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Select File",
-    "ja": "ファイルを選ぶ",
-    "fr": "Sélectioner le Fichier",
+    "ja": "\u30d5\u30a1\u30a4\u30eb\u3092\u9078\u3076",
```
```suggestion
    "ja": "ファイルを選ぶ",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Select File(s)",
-    "ja": "ファイルを選ぶ",
-    "fr": "Sélectioner le(s) Fichier(s)",
+    "ja": "\u30d5\u30a1\u30a4\u30eb\u3092\u9078\u3076",
```
```suggestion
    "ja": "ファイルを選ぶ",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "ファイルを選ぶ",
-    "fr": "Sélectioner le(s) Fichier(s)",
+    "ja": "\u30d5\u30a1\u30a4\u30eb\u3092\u9078\u3076",
+    "fr": "S\u00e9lectioner le(s) Fichier(s)",
```
```suggestion
    "fr": "Sélectioner le(s) Fichier(s)",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Select Project File",
-    "ja": "プロジェクトファイルを選択してください",
-    "fr": "Sélectioner le Fichier Projet",
+    "ja": "\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u30d5\u30a1\u30a4\u30eb\u3092\u9078\u629e\u3057\u3066\u304f\u3060\u3055\u3044",
```
```suggestion
    "ja": "プロジェクトファイルを選択してください",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_HOME_BUTTON": {
     "en": "Home",
-    "ja": "ホーム",
+    "ja": "\u30db\u30fc\u30e0",
```
```suggestion
    "ja": "ホーム",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_DOCUMENTS_BUTTON": {
     "en": "Documents",
-    "ja": "ドキュメント",
+    "ja": "\u30c9\u30ad\u30e5\u30e1\u30f3\u30c8",
```
```suggestion
    "ja": "ドキュメント",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "ダウンロード",
-    "fr": "Téléchargements",
+    "ja": "\u30c0\u30a6\u30f3\u30ed\u30fc\u30c9",
+    "fr": "T\u00e9l\u00e9chargements",
```
```suggestion
    "fr": "Téléchargements",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
   "FILE_EXPLORER_PICTURES_BUTTON": {
     "en": "Pictures",
-    "ja": "写真",
+    "ja": "\u5199\u771f",
```
```suggestion
    "ja": "写真",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Videos",
-    "ja": "動画",
-    "fr": "Vidéos",
+    "ja": "\u52d5\u753b",
```
```suggestion
    "ja": "動画",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
     "en": "Saved paths",
-    "ja": "保存されたパス",
-    "fr": "Chemins enregistrés",
+    "ja": "\u4fdd\u5b58\u3055\u308c\u305f\u30d1\u30b9",
```
```suggestion
    "ja": "保存されたパス",
```

voylin commented on translations/locales.json:
```diff
@@ -1,319 +1,354 @@
-    "ja": "保存されたパス",
-    "fr": "Chemins enregistrés",
+    "ja": "\u4fdd\u5b58\u3055\u308c\u305f\u30d1\u30b9",
+    "fr": "Chemins enregistr\u00e9s",
```
```suggestion
    "fr": "Chemins enregistrés",
```

voylin committed:
Apply fixes from adding new language

There is a bug in the python script which changes special characters (French, Chinese, and Japanese)

voylin commented on translations/locales.json:
```diff
@@ -4,316 +4,351 @@
     "en": "Settings",
     "ja": "設定",
-    "fr": "Réglages",
+    "fr": "R\u00e9glages",
```
```suggestion
    "fr": "Réglages",
```

voylin committed:
Apply Fix from adding language (small fix)

Forgot a word in French

voylin commented:
Merged! I'll fix the python script to avoid this whole character changing problem ^^"

voylin commented:
Thank you for your work! :D
